client/src/assets/icons/completion-ribbon.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import { userSelector } from '../../redux/selectors';
+import { Themes } from '../../components/settings/theme';
+
+const mapStateToProps = createSelector(
+  userSelector,
+  ({ theme = Themes.Default }: { theme: Themes }) => ({
+    theme
+  })
+);
+
+interface RibbonProps {
+  value: number;
+  theme: Themes;
+}
+
+const RibbonIcon = ({ value, theme }: RibbonProps): JSX.Element => {
+  const fillColor = theme === Themes.Night ? 'white' : 'black';
```

Comment:
This approach is unneeded, because the component isn't an external one that require a specific way to change its style.

Please change the style through CSS.